src/tokens/hooks.ts

```diff
@@ -84,13 +85,19 @@ export function useTokenBalance(token: TokenInfo, address: string, refresh?: boo
     queryKey: ["tokenBalance", token.processId, address],
     queryFn: async () => {
       try {
-        const balance = await fetchTokenBalance(token, address, refresh);
+        const balance = await retryWithDelay(
```

Comment:
Disabled tanstak retry as it was acting weird and not working as expected. So, used the custom retry inside queryFn and now if there is any error from fetchTokenBalance regarding network or balance fetch, it would be correctly shown with the error tooltips for tokens.